include/fiction/io/write_svg_layout.hpp

```diff
@@ -37,6 +43,58 @@ struct write_qca_layout_svg_params
     bool simple = false;
 };
 
+/**
+ * Enumeration to specify the color mode for the SVG output.
+ */
+enum class color_mode : uint8_t
+{
+    /**
+     * Dark mode.
+     */
+    DARK,
+    /**
+     * Light mode.
+     */
+    LIGHT
```

Comment:
Could you indicate here what exactly changes in the two modes? Background color, dot color, etc.

Reply:
Since we don't have it for QCA, you might as well move it into `write_sidb_layout_svg_params`.